client/src/App.tsx

```diff
@@ -147,6 +147,7 @@ function App() {
                       padding: var(--mi-side-paddings-xsmall);
```

Comment:
I feel like if there's a div that has consistent styles in all the <Outlet> spaces then the styles should be defined on those `div`s and not here, and the <div> should be turned into its own re-used component.

Reply:
You're right! That was actually something I wanted to do but didn't know how at the time but now that I've done the AlbumGrid Component I feel like it's fairly similar and I should be able to do it right?